app-backend/build.sbt

```diff
@@ -422,7 +424,7 @@ lazy val akkautil = project
   * Backsplash Core Settings
   */
 lazy val backsplashCore = Project("backsplash-core", file("backsplash-core"))
-  .dependsOn(common)
+  .dependsOn(common, db)
```

Comment:
I don't _think_ that I saw anything in `backsplash-core` that depends on the `db` -- is this leftover?

Reply:
This got ordered weirdly but there's a suggestion a tiny bit down (below the :sunglasses: comment) if you don't want to make this change manually

Reply:
Ah good point, I think it was a holdover